pkgs/by-name/wp/wpgtk/package.nix

```diff
@@ -55,9 +59,9 @@ python3Packages.buildPythonApplication rec {
       INFO: To work properly, this tool needs "programs.dconf.enable = true" on nixos or dconf installed. A reboot may be required after installing dconf.
     '';
     homepage = "https://github.com/deviantfero/wpgtk";
-    license = licenses.gpl2Only;
-    platforms = platforms.linux;
-    maintainers = with maintainers; [
+    license = lib.licenses.gpl2Only;
+    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [
```

Comment:
A minor nitpick would be that it would be ideal if refactoring like this was split from version upgrades, since it's not nescesarry to advance the version. But that's in no way blocking of course!